src/commonTest/kotlin/fr/acinq/lightning/wire/OpenTlvTestsCommon.kt

```diff
@@ -63,13 +63,13 @@ class OpenTlvTestsCommon : LightningTestSuite() {
     @Test
     fun `channel origin TLV`() {
         val testCases = listOf(
-            Pair(
+            /*Pair(
                 ChannelOrigin.PayToOpenOrigin(ByteVector32.fromValidHex("187bf923f7f11ef732b73c417eb5a57cd4667b20a6f130ff505cd7ad3ab87281"), 1234.sat),
                 Hex.decode("fe47000005 2a 0001 187bf923f7f11ef732b73c417eb5a57cd4667b20a6f130ff505cd7ad3ab87281 00000000000004d2")
-            ),
+            ),*/
```

Comment:
Oops, this needs to be uncommented, my bad!

Reply:
Done in [d9eb1ad](https://github.com/ACINQ/lightning-kmp/pull/395/commits/d9eb1adf48c6daa6708da13273b8bac64eabbc86)